keyboards/keebio/iris/rules.mk

```diff
@@ -8,7 +8,7 @@ F_USB = $(F_CPU)
 #     different sizes, comment this out, and the correct address will be loaded
 #     automatically (+60). See bootloader.mk for all options.
 ifeq ($(strip $(KEYBOARD)), iris/rev3)
```

Comment:
This line must be:
> `ifeq ($(strip $(KEYBOARD)), keebio/iris/rev3)`

because folder structure was changed.

Reply:
Thanks for pointing this out, I've submitted a fix here that makes the revision detection independent of folder structure: https://github.com/qmk/qmk_firmware/pull/6226